inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -71,8 +71,12 @@ def does_exist_in_inspirehep(query, collection=None):
         'of': 'id'
     }
 
-    if collection:
-        params['cc'] = collection
+    if len(kwargs) > 1:
+        collections = []
+        for key, value in kwargs.iteritems():
+            if 'collection' in key:
+                collections.append(value)
```

Comment:
To avoid this somewhat awkward logic (which leads to you having to have parameters called `collection1` and `collection2`) can't you just have `collection` be a list, and iterate over it?